commodore/catalog.py

```diff
@@ -124,3 +129,42 @@ def catalog_list(cfg):
             click.echo(f" - {display_name}")
         else:
             click.echo(catalog_id)
+
+
+def list_components(config: Config, cluster_id: str, output: Optional[str] = None):
+    """
+    List all components required by target. Generate list of components
+    by searching for classes with prefix `components.` in the inventory files.
```

Comment:
I assume you've copied the function comment from `fetch_components`. I just noticed that that comment is outdated.

Since #227, the list of components included for the cluster is extracted from the `applications` array as described in https://syn.tools/commodore/reference/architecture.html#_component_discovery_and_versions